taca/utils/misc.py

```diff
@@ -38,10 +38,9 @@ def call_external_command(cl, with_log_files=False, prefix=None):
     command = os.path.basename(cl[0])
     stdout = sys.stdout
     stderr = sys.stderr
-
     if with_log_files:
         if prefix:
-            command = '{}_{}'.format(prefix, command)
+            command = os.path.join('{}'.format(prefix),  '{}'.format(command))
```

Comment:
@vezzi : Is `prefix` as directory (when passed) and would it exists always ? If not, it would fail.